config/config.go

```diff
@@ -179,3 +188,18 @@ func GetOptionalParameters(config *Config, defaultLidarDataRateMsec, defaultIMUD
 
 	return optionalConfigParams, nil
 }
+
+func validateModes(
+	optionalConfigParams OptionalConfigParams,
+	imuEnabled bool,
+) error {
+	localizationMode := optionalConfigParams.LidarDataRateMsec == 0
```

Comment:
This seems wrong - isn't it that if LidarDataRateMsec == 0, we're in *offline* mode, not *localization* mode?

Reply:
yes got them backwards fixing!